SQL/0000-00-00-schema.sql

```diff
@@ -1643,6 +1671,386 @@ CREATE TABLE `parameter_type_override` (
+  `SpecimenTypeID` integer unsigned NOT NULL,                                   
+  `UnitID` integer unsigned  NOT NULL,                                          
+  CONSTRAINT `PK_biobank_container_psc_rel` PRIMARY KEY (`SpecimenTypeID`, `UnitID`),
+  CONSTRAINT `FK_biobank_specimen_type_unit_rel_TypeID`                         
+    FOREIGN KEY (`SpecimenTypeID`) REFERENCES `biobank_specimen_type` (`SpecimenTypeID`)
+    ON UPDATE RESTRICT ON DELETE RESTRICT,                                      
+  CONSTRAINT `FK_biobank_specimen_type_unit_rel_SourceID`                       
+    FOREIGN KEY (`UnitID`) REFERENCES `biobank_unit` (`UnitID`)                 
+    ON UPDATE RESTRICT ON DELETE RESTRICT                                       
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;                                        
+                                                                                
+CREATE TABLE `biobank_specimen_parent` (                                        
+  `SpecimenID` integer unsigned NOT NULL,                                       
+  `ParentSpecimenID` integer unsigned NOT NULL,                                 
+  CONSTRAINT `PK_biobank_specimen_parent` PRIMARY KEY (`SpecimenID`, `ParentSpecimenID`),
+  CONSTRAINT `FK_biobank_specimen_parent_SpecimenID`                            
+    FOREIGN KEY (`SpecimenID`) REFERENCES `biobank_specimen`(`SpecimenID`)      
+    ON UPDATE RESTRICT ON DELETE RESTRICT,                                      
+  CONSTRAINT `FK_biobank_specimen_parent_ParentSpecimenID`                      
+    FOREIGN KEY (`ParentSpecimenID`) REFERENCES `biobank_specimen`(`SpecimenID`)
+    ON UPDATE RESTRICT ON DELETE RESTRICT                                       
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;                                        
+                                                                                
+CREATE TABLE `biobank_specimen_pool_rel` (                                      
+  `SpecimenID` integer unsigned NOT NULL,                                       
+  `PoolID` integer unsigned NOT NULL,                                           
+  CONSTRAINT `PK_biobank_specimen_pool_rel` PRIMARY KEY (`SpecimenID`, `PoolID`),
+  CONSTRAINT `FK_biobank_specimen_pool_rel_SpecimenID`                          
+    FOREIGN KEY (`SpecimenID`) REFERENCES `biobank_specimen`(`SpecimenID`)      
+    ON UPDATE RESTRICT ON DELETE RESTRICT,                                      
+  CONSTRAINT `FK_biobank_specimen_pool_rel_PoolID`                              
+    FOREIGN KEY (`PoolID`) REFERENCES `biobank_pool`(`PoolID`)                  
+    ON UPDATE RESTRICT ON DELETE RESTRICT                                       
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;                                        
+                                                                                
+CREATE TABLE `biobank_container_parent` (                                       
```

Comment:
Maybe rename to `biobank_container_coordinate`

Reply:
I think `biobank_container_parent` is more appropriate because a container can be associated with a parent and have a `Coordinate` of null. The important relationship that this rel table establishes is that between a child container and a parent container.

Reply:
ok